Composer/packages/client/src/pages/botProject/adapters/ExternalAdapterModal.tsx

```diff
@@ -15,17 +15,12 @@ import { JSONSchema7 } from '@botframework-composer/types';
 import { EditorExtension, PluginConfig } from '@bfc/extension-client';
 import mapValues from 'lodash/mapValues';
 import { JSONSchema7Type } from 'json-schema';
+import { AdapterRecord } from '@botframework-composer/types/src';
```

Comment:
This should not import from the src of the types package.

Reply:
Handled